Core/Core/Inbox/ComposeMessage/View/ComposeMessageView.swift

```diff
@@ -47,7 +47,6 @@ public struct ComposeMessageView: View, ScreenViewTrackable {
 
     public var body: some View {
         InstUI.BaseScreen(state: model.state, config: model.screenConfig) { geometry in
-            ScrollView {
```

Comment:
We already have a scrollView inside  InstUI.BaseScreen so we no longer need it.

Reply:
Nice catch!